packetbeat/docs/packetbeat-options.asciidoc

```diff
@@ -461,6 +461,12 @@ in time. Periodical reporting can be disabled by setting the value to -1. If
 disabled, flows are still reported once being timed out. The default value is
 10s.
 
+[float]
+==== `enable_active_flow_timeout`
```

Comment:
same point - I think doc needs to be updated?

Reply:
Added comments, hope it helps. I can look at making it more understandable if not.